### ISSUE TITLE
Dates not being returned

### ISSUE THREAD

**Issue by sihartley** [pinkywafer/Calendarific#24]
For some reason the dates are not being returned, when I pull the state  for a date I get the following...

date: '-'
description: NOT FOUND
attribution: Data provided by calendarific.com
unit_of_measurement: Days
friendly_name: Independence Day
icon: mdi:calendar-blank

**Comment by Shaun-Harrison** [comment]
I'm also getting this - on start up can see the integration is saying that i've exceeded my usage limits, even though i've used 2/1000 at the moment
Any ideas @pinkywafer

**Comment by pinkywafer** [comment]
I'm afraid I've seen the same in the past when doing many restarts and have hit the api limit. - I have not been able to find any way of resetting the usage or find out how long it takes to reset and when I have contacted the calendarific team through their website, I have not received an answer. The only work arounds I have found are either 1. (not recommended!) sign up for a paid account at $10/month. or 2. Sign up for a new free account with a different email address.
Sorry I can't do any more with this, but I am not directly associated with the Calendarific team and have been unable to contact them myself also.
